src/main/java/hng_java_boilerplate/organisation/dto/InvitationRequest.java

```diff
@@ -14,14 +14,14 @@ public class InvitationRequest {
     @Autowired
     private OrganisationService organisationService;
```

Comment:
Avoid field injection, also do you need the service in a dto?
Use data annotation